src/SablierV2MerkleLT.sol

```diff
@@ -46,12 +51,14 @@ contract SablierV2MerkleLT is
     {
         LOCKUP_TRANCHED = lockupTranched;
 
-        // Since Solidity lacks a syntax for copying arrays of structs directly from memory to storage, a manual
-        // approach is necessary. See https://github.com/ethereum/solidity/issues/12783.
-        uint256 count = tranchesWithPercentages.length;
-        for (uint256 i = 0; i < count; ++i) {
+        // Calculate the total percentage of the tranches and save them in the contract storage.
+        uint64 totalPercentage;
```

Comment:
let's say contract state, as `TOTAL_PERCENTAGE` is immutable and it is a part of the bytecode

Reply:
https://github.com/sablier-labs/v2-periphery/pull/355/commits/013c5dee0a7aeb9611182b20f9283cc94f4e5daf